server/gateway/config.json

```diff
@@ -82,10 +82,8 @@
         "blobStorageUrl": "http://historian:3000",
         "internalBlobStorageUrl": "http://historian:3000",
         "repository": "prague",
-        "npm": "https://pragueauspkn.azureedge.net",
-        "clusterNpm": "https://pragueauspkn.azureedge.net",
-        "localNpm": "http://localhost:3002",
-        "localClusterNpm": "http://auspkn:3000",
+        "npm": "https://unpkg.com",
+        "clusterNpm": "https://unpkg.com",
```

Comment:
Do we need both "npm" and "clusterNpm"? If so, then kube configmap.yaml need that as well?

Reply:
I believe it is not in the kube configmap. We should probably trim the config file to actually represent the used values. I suspect we have superfluous entries.

Reply:
I am sure they do. Having npm and clusterNpm both points to the same thing looks weird!